internal/cache/live_test.go

```diff
@@ -0,0 +1,63 @@
+//go:build go1.24
+
+package cache
+
+import (
+	"context"
+	"encoding/binary"
+	"runtime"
+	"testing"
+	"weak"
+)
+
+func TestLive(t *testing.T) {
+	var c Live[uint32, [4]byte]
+	c.Create = func(_ context.Context, key uint32) (*[4]byte, error) {
+		var v [4]byte
+		binary.LittleEndian.PutUint32(v[:], key)
+		return &v, nil
+	}
+	ctx := t.Context()
+
+	a, err := c.Get(ctx, 0xF000000F, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	b, err := c.Get(ctx, 0xF000000F, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	wp := weak.Make(b)
+
+	t.Logf("a: %p, b: %p", a, b)
+	if a != b {
+		t.Fail()
+	}
+	a, b = nil, nil
+
+	for range 2 { // Need two cycles:
```

Comment:
Do you know the reason?

Reply:
I double checked this out of curiosity, and it looks like a single `runtime.GC()` call works on `go version go1.25.0 darwin/arm64`